### DISCUSSION TITLE
use with flax (and other forks)

### DISCUSSION THREAD

**Discussion by atrebbi** [martomi/chiadog#344] [General]
can I use the tool with flax ? I would like to monitor flax as well

**Comment by martomi** [comment]
There's a fork with that implementation but it's not up-to-date with master anymore. See it here: https://github.com/martomi/chiadog/pull/302

**Reply by atrebbi** [reply]
I see the PR, why wasn't it merged ?

**Reply by martomi** [reply]
If I remember correctly, my main concern was that it would open an entire can of worms for maintenance and support.

We're relying on very specific formatting of the debug logs for all chiadog's functionality. This is not an official or fixed API contract. As time progresses, any of these unofficially supported forks might introduce backwards incompatible changes in their logging that are generally diverging from the logging format used in the official chia software. 

I don't want to start getting support questions and bugs for every single of these forks when such events occur. It'll also make maintenance increasingly more time consuming and error-prone to keep compatibility with a large number of these forks.

If someone else is willing to put in that effort and commit to the maintenance that comes with it, I'm happy to link to a forked version of chiadog that provides this support. It'll be up to the maintainer of the fork to keep up-to-date with the upstream chiadog and address any support questions for chia forks. The former should be fairly easy as this repository is now in maintenance mode and no new features are planned.

**Reply by martomi** [reply]
Here's an example of a commit that I don't want to have to do in this repository. 
https://github.com/guydavis/chiadog/commit/d3ee7c3d70daa7b3c154cdafb9b4665a35a3e60b

That being said, the fork from @guydavis is not terribly outdated so you could try it out on your own risk.
